src/services/factories/getTokenFromErc20.ts

```diff
@@ -14,10 +15,8 @@ export interface TokenFromErc20Params {
   networkId: number
 }
 
-export type TokenFromErc20 = Omit<TokenDetails, 'id'> | null
-
 export function getTokenFromErc20Factory(closureParams: Params) {
-  return async (params: TokenFromErc20Params): Promise<TokenFromErc20> => {
+  return async (params: TokenFromErc20Params): Promise<TokenErc20 | null> => {
```

Comment:
Why wan't this `| null` before?

Reply:
It was https://github.com/gnosis/dex-react/pull/1454/files#diff-dfb2c72865d565e8a1e493c4f13fbce1L17

it was just a bit more obscure declaration